src/style.css

```diff
@@ -0,0 +1,29 @@
+/* stylelint-disable CssSyntaxError */
```

Comment:
Remove comment 
If error still consist try to fix it
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review